test/extended/etcd/etcd_storage_path.go

```diff
@@ -278,19 +278,8 @@ func testEtcd3StoragePath(t g.GinkgoTInterface, kubeConfig *restclient.Config, e
 		gvr("scheduling.k8s.io", "v1alpha1", "priorityclasses"),
 		gvr("settings.k8s.io", "v1alpha1", "podpresets"),
 		gvr("storage.k8s.io", "v1alpha1", "volumeattachments"),
-		gvr("extensions", "v1beta1", "deployments"),
-		gvr("extensions", "v1beta1", "podsecuritypolicies"),
-		gvr("extensions", "v1beta1", "replicasets"),
-		gvr("extensions", "v1beta1", "daemonsets"),
-		gvr("extensions", "v1beta1", "networkpolicies"),
-		gvr("apps", "v1beta2", "deployments"),
-		gvr("apps", "v1beta2", "replicasets"),
-		gvr("apps", "v1beta2", "controllerrevisions"),
-		gvr("apps", "v1beta2", "statefulsets"),
-		gvr("apps", "v1beta2", "daemonsets"),
-		gvr("apps", "v1beta1", "controllerrevisions"),
-		gvr("apps", "v1beta1", "statefulsets"),
-		gvr("apps", "v1beta1", "deployments"),
```

Comment:
@soltysh this is expected for apps?

Reply:
According to the following upstream commit, these gvr's are no longer served:
 https://github.com/openshift/kubernetes/commit/18ba6f94820976e617755135abe9d0e7d4192229#diff-a6932c2bd6398cd55df3be4564939a5e

Reply:
Yes, that is correct.